application/controllers/ContactGroupController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/* Icinga Notifications Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Notifications\Controllers;
+
+use Icinga\Module\Notifications\Common\Database;
+use ipl\Sql\Connection;
+use ipl\Web\Compat\CompatController;
+
+class ContactGroupController extends CompatController
+{
+    /** @var Connection $db */
+    private $db;
```

Comment:
As `Database::get()` returns a singleton instance, you can use the method directly.